pandas/tests/io/formats/test_style.py

```diff
@@ -1335,35 +1342,67 @@ def test_display_dict(self):
 
     def test_bad_apply_shape(self):
         df = pd.DataFrame([[1, 2], [3, 4]])
-        with pytest.raises(ValueError):
+        msg = (
+            "Function <function TestStyler.test_bad_apply_shape.<locals>.<lambda> "
+            "at 0x[a-zA-Z0-9]+> returned the wrong shape.\n"
+            "Result has shape: \\(2,\\)\nExpected shape:   \\(2, 2\\)"
```

Comment:
Can you simplify those messages? (same for the ones below) Asserting part of it to avoid many regexes should be fine I think

Reply:
Maybe to this?
```py
msg = (
            "Function <function TestStyler.test_bad_apply_shape.<locals>.<lambda> "
            "at 0x[a-zA-Z0-9]+> returned the wrong shape.\n"
        )
```
I think the function name and the returning wrong shape part are still important.
Or what you mean is just
```py
msg = "returned the wrong shape."
```

Reply:
I think the last short one is fine